recipe/build.sh

```diff
@@ -1,4 +1,6 @@
-$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+#!/bin/bash
+
+$PYTHON -m pip install . --no-deps -vv
```

Comment:
Same style as in https://github.com/conda-forge/bluesky-feedstock/blob/f2f618c53741e2b1131b51af41639c4284e48cf4/recipe/meta.yaml#L16, also [recommended](https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip) by CF.